action.yml

```diff
@@ -11,8 +11,8 @@ inputs:
   zulip_bot_key:
     description: 'API key of the Zulip bot'
     required: true
-  github_personal_access_token:
-    description: 'GitHub personal access token'
+  gh_token:
```

Comment:
This will be a breaking change, but I think it is fine as some users already pinned their zulip-archive version to certain commits